packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.tsx

```diff
@@ -1,196 +1,9 @@
-import * as React from 'react';
+import { styled } from '../../Utilities';
```

Comment:
This should be a new file called Searchbox.ts, as opposed to tsx, so history is preserved.

Reply:
oh shoot....I think I put the wrong item into the milestone